editor/src/components/canvas/remix/remix-navigator.spec.browser2.tsx

```diff
@@ -47,7 +47,7 @@ describe('Remix navigator', () => {
         </Storyboard>
       )
       `,
-      ['/src/root.js']: `import React from 'react'
+      ['/app/root.js']: `import React from 'react'
```

Comment:
is it true that in this test you could keep the test routes `/src` and include a `remix.config.js` among the test project files?

Reply:
it's true, and I included such a test (called `Parses different route dir`, which had to be fixed in https://github.com/concrete-utopia/utopia/pull/4450/commits/063843a046f927a9d8ed27a9a075b47b86fd70cf).

I renamed `src` to `app` in the existing tests because including the remix config file would have been basically the same diff, but with more context, and using `app` by default reflects that `app` is the default folder